### PR TITLE
Use filters with NeoFS responses

### PR DESCRIPTION
**Pull request by fyrchik**
See also https://github.com/neo-project/neo-modules/issues/555 .

### CODE REVIEW

**Comment by roman-khimov** [comment]
Waiting for neo-project/neo-modules#555 and neo-project/neo-modules#554.

File: pkg/services/oracle/neofs/neofs.go [hunk 1]
```diff
@@ -149,7 +155,7 @@ func getHash(ctx context.Context, c *client.Client, addr *object.Address, ps ...
 	if len(hashes) == 0 {
 		return nil, fmt.Errorf("%w: empty response", ErrInvalidRange)
 	}
-	return hashes[0][:], nil
+	return util.Uint256(hashes[0]).MarshalJSON()
```
Comment (roman-khimov) [inline pkg/services/oracle/neofs/neofs.go:158]: It's wrapped into object which `MarshalJSON()` won't do IIRC.

Comment (fyrchik) [reply pkg/services/oracle/neofs/neofs.go:158]: It is string:
https://github.com/neo-project/neo-modules/blob/master/src/OracleService/Protocols/OracleNeoFSProtocol.cs#L119

Comment (roman-khimov) [reply pkg/services/oracle/neofs/neofs.go:158]: OK, I guess I can't read C# interpolated strings properly.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**